handbook/engineering/campaigns/index.md

```diff
@@ -82,6 +70,18 @@ Our users use more than just their code host and Sourcegraph. They use ticket tr
 
 We want to discover what external systems our users want to use campaigns with, and ensure that we can integrate with them. For example, an organization that uses JIRA will likely want to be able to link tickets to campaigns and have the state updated as the campaign is executed.
 
+## Roadmap
+
+Our team's public roadmap is part of our larger [Product roadmap](https://about.sourcegraph.com/handbook/product/roadmap#campaigns).
```

Comment:
@christinaforney We are moving toward inlining roadmaps in team pages, right?

Reply:
I'm going to go ahead and merge this, but am happy to move things around later based on feedback here.

Reply:
Yes. Updated for all teams here: https://github.com/sourcegraph/about/pull/1984